tests/system/action/meeting/test_import.py

```diff
@@ -23,6 +23,158 @@ def create_request_data(self, datapart: Dict[str, Any]) -> Dict[str, Any]:
                         "projector_countdown_warning_time": 60,
                         "reference_projector_id": 1,
                         "user_ids": [1],
+                        "imported_at": None,
```

Comment:
Can you use https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/docs/example-data.json [1], load it once and only do your modifications for each test?

[1] You can copy this file next to the test and remove unwanted collections (organization, committee, ...)

Reply:
No, that would be not good. The tests work by the way.